example/src/App.tsx

```diff
@@ -65,6 +65,16 @@ const App = () => {
 
     logStatus('SDK Initialized');
 
+    if (Platform.OS === 'android') {
```

Comment:
We should probably follow our own docs:

https://dash.applovin.com/documentation/mediation/react-native/getting-started/privacy#general-data-protection-regulation-(%E2%80%9Cgdpr%E2%80%9D)

Reply:
Ok, I followed but changed the code style for React Native.

Reply:
That's fine